### PR TITLE
Complete traceability matrix

### PR DESCRIPTION
**Pull request by Veykril**

### CODE REVIEW

**Comment by pietroalbini** [comment]
Tidy is complaining:

```
tidy error: /home/ci/project/tests/ui/simd/const-err-trumps-simd-err.rs: too many trailing newlines (2)
```

File: tests/ui/const-generics/generic_const_exprs/const_kind_expr/relate_ty_with_infer_2.rs
Comment (tshepang) [inline tests/ui/const-generics/generic_const_exprs/const_kind_expr/relate_ty_with_infer_2.rs:1]: is there a const block in this file

File: tests/ui/lifetimes/unusual-rib-combinations.rs
Comment (tshepang) [inline tests/ui/lifetimes/unusual-rib-combinations.rs:1]: is there a const block in this file

File: tests/ui/lint/invalid_from_utf8.rs [hunk 1]
```diff
@@ -1,7 +1,6 @@
 //@ check-pass
 
 #![feature(concat_bytes)]
-
```
Comment (tshepang) [inline tests/ui/lint/invalid_from_utf8.rs:4]: deviates from upstream

File: tests/ui/lint/invalid_from_utf8.rs
Comment (tshepang) [inline tests/ui/lint/invalid_from_utf8.rs:1]: is there a const block in this file

**Comment by Veykril** [comment]
Incorrect const block annotations came from me just applying the annotation to every file that used to have `feature(inline_const)`, turns out that feature was incorrectly placed in some files I suppose

**Review by tshepang — APPROVED** [review]
bors merge

**Pull request merged**